examples/fluids/src/misc.c

```diff
@@ -257,3 +258,29 @@ int FreeContextPetsc(void *data) {
   if (PetscFree(data)) return CeedError(NULL, CEED_ERROR_ACCESS, "PetscFree failed");
   return CEED_ERROR_SUCCESS;
 }
+
+// Return mass qfunction specification for number of components N
+PetscErrorCode ReturnMassQFunction(CeedInt N, ProblemQFunctionSpec *qf_spec) {
```

Comment:
Could we make this return a `CeedQFunction`?
```suggestion
PetscErrorCode CreateMassQFunction(Ceed ceed, CeedInt N, CeedQFunction *qf) {
```

Reply:
Ah, yeah, that'd be cleaner.